lifting_veil/agents/rainbow_agent_new.py

```diff
@@ -322,10 +331,12 @@ def _train_step(self):
         else:
           loss_weights = jnp.ones(self.replay_elements['state'].shape[0])
 
-        self.optimizer, loss, mean_loss = train(
+        (self.optimizer_state, self.online_params, loss, mean_loss) = train(
```

Comment:
you don't need the parentheses on the left-hand side. i may have used them in the dopamine code base because there was a newline in the middle.

Reply:
Perfect!